app/assets/javascripts/discourse/app/components/footer-nav.gjs

```diff
@@ -0,0 +1,143 @@
+import Component from "@glimmer/component";
+import { action } from "@ember/object";
+import { service } from "@ember/service";
+import DButton from "discourse/components/d-button";
+import concatClass from "discourse/helpers/concat-class";
+import htmlClass from "discourse/helpers/html-class";
+import { postRNWebviewMessage } from "discourse/lib/utilities";
+import { SCROLLED_UP, UNSCROLLED } from "discourse/services/scroll-direction";
+import not from "truth-helpers/helpers/not";
+
+class FooterNav extends Component {
+  @service appEvents;
+  @service capabilities;
+  @service scrollDirection;
+  @service composer;
+  @service modal;
+  @service historyStore;
+
+  _modalOn() {
+    const backdrop = document.querySelector(".modal-backdrop");
+    if (!backdrop) {
+      return;
+    }
+
+    postRNWebviewMessage(
+      "headerBg",
+      getComputedStyle(backdrop)["background-color"]
+    );
+  }
+
+  _modalOff() {
+    const dheader = document.querySelector(".d-header");
+    if (!dheader) {
+      return;
+    }
+
```

Comment:
```suggestion
    const dheader = document.querySelector(".d-header");
    if (!dheader) {
      return;
    }

```
we don't need the header to be present any more. We're using document.documentElement instead, which will always be there

Reply:
oops yes